run.sh

```diff
@@ -9,26 +9,8 @@ ENV_ERROR="$LOG_PREFIX Error: this module requires Python >=3.8.1, pip3, and vir
 
 if ! python3 -m venv "$VENV_NAME" >/dev/null 2>&1; then
     echo "$LOG_PREFIX Warning: failed to create virtualenv."
-    if command -v apt-get >/dev/null; then
```

Comment:
I think this is a good call to remove hot system installs here and use error message to prompt user.